aten/src/ATen/cpu/vec256/vec256_double.h

```diff
@@ -238,6 +238,19 @@ Vec256<double> inline operator^(const Vec256<double>& a, const Vec256<double>& b
   return _mm256_xor_pd(a, b);
 }
 
+template <>
+void convert(const double* src, double* dst, int64_t n) {
+  int64_t i;
+#pragma unroll
+  for (i = 0; i <= (n - Vec256<double>::size); i += Vec256<double>::size) {
+    _mm256_storeu_pd(dst + i, _mm256_loadu_pd(src + i));
```

Comment:
You can reuse the loadu/storeu methods on this class to avoid having to reimplement the boundary condition. Doesn't have to be this patch but might be worth cleaning up slightly later.

Also CC @cpuhrsch

Reply:
...though, I guess you got this pattern from the existing implementations of convert? So maybe it's all fine. (cc @SsnL too, if you want to weigh in.)